sonic-net/sonic-buildimage
# [Nokia-7220-H3] Add support for NOKIA 7220 IXR-H3 platform

#### Why I did it
Add support for NOKIA 7220 IXR-H3:

>     Platform: x86_64-nokia_ixr7220_h3-r0
>     HwSKU: Nokia-IXR7220-H3
>     ASIC: Broadcom
>     Port Config: 32x400G + 2x10G

##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it
1.	Added module x86_64-nokia_ixr7220_h3-r0 ../device/nokia directory.
2.	Modified the platform/broadcom/one-image.mk to include NOKIA_IXR7220H3_PLATFORM_MODULE
3.	Modified the platform/broadcom/ platform-modules-nokia.mk to include NOKIA_IXR7220H3_PLATFORM_MODULE
4.	Add platform related directory ixr7220h3 in platform/Broadcom/sonic-platform-modules-nokia
5.	Modified related build scripts in platform/Broadcom/sonic-platform-modules-nokia/debian

#### How to verify it
1.	Make sure the sonic-buildimage is successful 
2.	Run this image on x86_64-nokia_ixr7220_h3-r0 and verify all dockers are up and test basic commands like:
                  o	show version
                  o	show platform summary
                  o	show platform syseeprom
                  o	show platform fan
                  o	show platform psustatus
                  o	show platform firmware status
                  o	show platform temperature
                  o	sudo show system-health detail
                  o	show interface status
3.	Run OC test, for T0/T1 topology and have 100% pass on platform.
4.	Run this image on x86_64-nokia_ixr7250e_sup-r0 and x86_64-nokia_ixr7250e_36x400g-r0, make sure all functions run correctly.



#### Which release branch to backport (provide reason below if selected)




#### Tested branch (Please provide the tested image version)




#### Description for the changelog




#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
